scripts/devservices-healthcheck.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This is used for healthchecking because docker running healthchecks
+# keeps the CPU too warm and if a container becomes unhealthy, there's no
+# notification system or anything that would make it more easily visible.
+
+# In the future, more healthchecks should be added to make it more helpful
+# as a troubleshooting script. Right now this is just used by CI.
+
+# TODO: iterate through all sentry_ containers,
+#       and warn on anything that's expected to run but isn't running.
+# TODO: spawn subprocess for each container,
+#       specify distinct retries and other parameters.
+#       would also be nice to have a single ^C work.
```

Comment:
All of the above is very much appreciated and lines up well with some of my concerns. Thanks for taking the time to lay it out.